packages/reports/app/styles/navi-reports/components/filter-collection.less

```diff
@@ -30,7 +30,7 @@
     }
 
     &__select-trigger {
-      border: 1px solid @navi-gray-400;
+      border: 1px solid #aaaaaa;
```

Comment:
We should probably make this color a variable somewhere. I think we've used it for border colors in the past too.